<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>826</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="graphicwindow" name="widget" native="true">
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="3">
       <widget class="QPushButton" name="mov_z_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6" colspan="2">
       <widget class="QLineEdit" name="rot_z_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Тип:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="mov_z_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&lt;b&gt;Настройка ребер&lt;b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QPushButton" name="rib_color">
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;b&gt;Настройка вершин&lt;b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;___ Y ___&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="mov_x_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="6" colspan="2">
       <widget class="QLineEdit" name="rot_x_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="mov_x_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="rot_x_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="rib_size">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="mov_y_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="mov_x_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="gif_but">
        <property name="text">
         <string>GIF</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6" colspan="2">
       <widget class="QLineEdit" name="rot_y_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="6" colspan="3">
       <widget class="QComboBox" name="vert_type">
        <item>
         <property name="text">
          <string>Отсутствует</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Круг</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Квадрат</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;___ X ___&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QPushButton" name="rot_z_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="rot_z_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="mov_z_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="zoom_out">
        <property name="text">
         <string>-</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../misc/images/zoom-out.png</normaloff>../../misc/images/zoom-out.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="back_color">
        <property name="text">
         <string>Цвет фона</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="zoom_in">
        <property name="text">
         <string>+</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../misc/images/zoom-in.png</normaloff>../../misc/images/zoom-in.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QPushButton" name="rot_y_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Тип проекции:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QComboBox" name="rib_type">
        <item>
         <property name="text">
          <string>Сплошная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Пунктирная</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="6" colspan="3">
       <widget class="QPushButton" name="choose_button">
        <property name="text">
         <string>Выбрать файл</string>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Размер:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="screen_but">
        <property name="text">
         <string>Screen</string>
        </property>
       </widget>
      </item>
      <item row="7" column="7">
       <widget class="QSpinBox" name="vert_size">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;b&gt;Вращение&lt;b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="mov_y_value">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="rot_y_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Масштаб&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Вид:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="6" colspan="2">
       <widget class="QPushButton" name="vert_color">
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;___ Z ___&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="6">
       <widget class="QLineEdit" name="file_path_label">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="rot_x_min">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="mov_y_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="proect_type">
        <item>
         <property name="text">
          <string>Параллельная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Центральная</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;b&gt;Перемещение&lt;b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Толщина:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Кол-во V:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QComboBox" name="img_format">
        <item>
         <property name="text">
          <string>.bmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.jpeg</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Кол-во F:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="v_count">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="4" colspan="2">
       <widget class="QLineEdit" name="f_count">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>graphicwindow</class>
   <extends>QWidget</extends>
   <header location="global">graphicwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
